cypress/integration/modal.spec.ts

```diff
@@ -0,0 +1,39 @@
+import * as commands from "../support/commands";
+
+context("modal", () => {
+  beforeEach(() => {
+    commands.resetProxyState();
+    commands.onboardUser();
+    cy.visit("./public/index.html");
+  });
+
+  it("no modal stacking possible", () => {
```

Comment:
When writing tests I always follow the [rspec convention](https://www.betterspecs.org/#should). And the [Cypress examples](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Test-Structure) seem to follow it as well.

It says that a spec should read like a sentence, eg: "It disallows modal stacking", where "it" refers to the subject (context/describe) that is being tested.

```suggestion
  it("disallows modal stacking", () => {
```

IMO this leads to more readable output:

<img width="232" alt="Screenshot 2021-03-03 at 15 23 53" src="https://user-images.githubusercontent.com/158411/109819814-badb2600-7c34-11eb-83ae-695167b729ed.png">

vs

<img width="417" alt="Screenshot 2021-03-03 at 15 26 58" src="https://user-images.githubusercontent.com/158411/109820023-e827d400-7c34-11eb-86c5-ebf929af3216.png">